### ISSUE TITLE
Baseline: Tile2Vec

### ISSUE THREAD

**Issue by porterjenkins** [porterjenkins/region-representation-learning#15]
This one may be a bit trickier, as it is a very new model (AAAI'19) and is not 'off-the-shelf'

paper: https://arxiv.org/pdf/1805.02855.pdf
code (?): https://github.com/ermongroup/tile2vec

Try and use their code if possible. If not, we can probably use some of our AutoEncoder code + the triplet loss function the define in their paper.

1. Train a Tile2Vec model.
2. Extract the hidden representation. Write to embedding directory
3. Import the hidden representation to the file experiments/run_predict_job.py.
4. Use hidden representation as features in an xgboost model.
5. Evaluate k-fold CV error

**Comment by porterjenkins** [comment]
Nice work
